hotel_booking.py

```diff
@@ -0,0 +1,210 @@
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+import numpy as np
```

Comment:
I have strong opinions about using abbreviations instead of whole words. Not a good idea. If I am deep down in this code I have to scroll all the way to the top to know what pd is, instead of it just saying panda. Using short names saves the writer milliseconds but the reader looses seconds each time.